og_ui/src/Form/AdminSettingsForm.php

```diff
@@ -133,6 +132,13 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       }
     }
 
+    $form['auto_add_owners'] = [
```

Comment:
How about a more explicit name, like

```suggestion
    $form['auto_add_membership_to_group_owner'] = [
```

Reply:
Renamed to `auto_add_group_owner_membership`